src/protocols/rdp/rdp_settings.c

```diff
@@ -775,6 +783,14 @@ guac_rdp_settings* guac_rdp_parse_args(guac_user* user,
     settings->sftp_directory =
         guac_user_parse_args_string(user, GUAC_RDP_CLIENT_ARGS, argv,
                 IDX_SFTP_DIRECTORY, NULL);
+
+    /* Default keepalive value */
+    settings->sftp_keepalive =
+        guac_user_parse_args_int(user, GUAC_RDP_CLIENT_ARGS, argv,
+                IDX_SFTP_KEEPALIVE, 0);
+    if (settings->sftp_keepalive == 1)
```

Comment:
Well, 0 is also a valid value for disabling keepalives, so <= 1 wouldn't work, I don't think.  However, it is probably also good to check for values < 0 and avoid those.